acquia_lift.admin.wizard.inc

```diff
@@ -577,34 +685,78 @@ function acquia_lift_personalize_campaign_wizard_targeting_submit(&$form, &$form
     return;
   }
   module_load_include('inc', 'acquia_lift', 'acquia_lift.admin');
+  $targeting = array();
+  $audience_count = 0;
+  // Determine the values for the "Everyone else" audience
+  $else = array(
```

Comment:
A minor point but I think this variable name can lead to confusion, because when you're reading the code and hearing it in your head you're hearing things like "else weight equals blah", which... yeah (or maybe it's just me? I read code like that :))